### ISSUE TITLE
[Prometheus/Exemplar UI] Hide leading zeros in traceid

### ISSUE THREAD

**Issue by tonychoe** [grafana/grafana#45576]
**Describe the bug**
When visualizing the exemplars generated from the OpenTelemetry collector's spanmetrics processor, there are leading zeros in the traceid. It looks ugly and hard to read. This issue was discussed at https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/7928 and was concluded that removing zeros is better to be done by Grafana UI. Can you please hide the zeros?

**Steps to reproduce**
1. Configure the opentelemetry collector with spanmetrics processor, prometheus exporter, and prometheusremotewrite exporter.
2. Search and visualize the resulting metrics in Grafana e.g.
3. histogram_quantile(.90, sum(rate(latency_bucket{service_name="app"}[1m])) by (le))
4. Hover on the exemplars to see the pop up.
5. The traceid in the popup has leading zeros.

**What did you expect to see?**
No leading zeros in the traceid.

**What did you see instead?**
Leading zeros in the traceid.
<img width="1492" alt="leading zero" src="https://user-images.githubusercontent.com/12507518/154623275-5e34ebb7-c66e-47ec-9591-ae8451ba64c3.png">

**Environment**
Grafana 8.3
Prometheus 2.31.2
Tempo 1.3
opentelemetry-collector-contrib:0.44.0

**Comment by ifrost** [comment]
@grafana/observability-logs-and-traces / @connorlindsey  is it something you wanted to have a look at?

**Comment by connorlindsey** [comment]
If someone from metrics can take it on, that'd be a big help. I'm not sure exactly who 'owns' exemplars though. I can also take it if needed.

**Comment by toddtreece** [comment]
@tonychoe does tempo expect a trace ID with leading zeros?

**Comment by aocenas** [comment]
So not sure how simple this is, probably needs some investigation. Seems to happen only with open telemetry collector and I assume it does not prevent the link to work @tonychoe correct me if I am wrong.

At the same time not sure if we can count on knowing the format, could there be traceId from some other system that requires the leading zeros?

**Comment by tonychoe** [comment]
@toddtreece No, Tempo doesn't care about leading zeros. It works well when with leading zeroes or no leading zero.

 @aocenas Correct, the leading zeros only happens to the traces generated from OpenTelemetry Collector.
